ASCIT/donut
# Port to Python3.6

### Summary
- Start using Python3.6 to avoid future headaches.
- Used 2to3 to convert
  - `2to3 donut/`
  - `2to3 -w donut/` writes the changes and backs up the old files to `*.bak`
- Switched to pymysql

### Test Plan
- Unit Tests
  - Added DK's unit tests for `test_auth_utils.py`
  - Restructured test directories
- Regression Testing
  - Make an account
  - Try to make an account after an account already exists (should give error message).
  - Login / logout
  - Reset password
